docs/contributing/dependencies/index.md

```diff
@@ -55,12 +92,165 @@ version is **13 days**, and **13%** of repositories have adopted this version. R
 change. For more details read
 [Renovate documentation about Merge Confidence](https://docs.renovatebot.com/merge-confidence/).
 
-## Workflow
+## Management strategies
+
+:::success Our Goal: A sustainable review cadence
+
+The overarching purpose behind our use of these strategies is ensuring that our dependencies are as
+up-to-date as possible, while opening PRs for review at a frequency and level of complexity that is
+sustainable for all responsible teams.
 
 Renovate is currently scheduled to automatically create pull requests every 2 weeks. The goal of our
 dependency management process is for the teams to review and merge the opened pull requests in the
 same 2-week cadence. To avoid a large backlog of PRs and out-of-date packages accumulating.
 
+:::
+
+### Dependency Pinning Strategies
+
+#### Pinned Dependencies
+
+:::info Why do we pin dependencies?
+
+For background on why we pin, see [Renovate's](https://docs.renovatebot.com/dependency-pinning/)
+documentation.
+
+:::
+
+For our client projects which are not intended to be consumed as libraries, our strategy is to pin
+all dependencies.
+
+In addition, we use `package-lock.json` and `Cargo.lock` lock files, as recommended by Renovate.
```

Comment:
```suggestion
In addition, we use `package-lock.json` and `Cargo.lock` lock files, as recommended by Renovate.
```

⛏️ This means to combine the two paragraphs.

Reply:
Addressed with 365f5f1b9a44b24aa3abfe773768e992112cdc91.